common/randombytes.h

```diff
@@ -1,13 +1,26 @@
-#ifndef RANDOMBYTES_H
-#define RANDOMBYTES_H
-#include <stdint.h>
+#ifndef sss_RANDOMBYTES_H
```

Comment:
Why is this `sss_` now?

Reply:
Because that's how I copied it from upstream, and how @dsprenkels had it. We could change it to `RANDOMBYTES_H` though.